pkgs/development/libraries/libgpiod/default.nix

```diff
@@ -22,14 +22,19 @@ stdenv.mkDerivation rec {
     autoconf-archive
     pkg-config
     autoreconfHook
-  ];
+  ] ++ lib.optionals enablePython [ python3 ];
 
   configureFlags = [
     "--enable-tools=${if enable-tools then "yes" else "no"}"
     "--enable-bindings-cxx"
     "--prefix=${placeholder "out"}"
   ] ++ lib.optional enablePython "--enable-bindings-python";
 
+  postInstall = lib.optional enablePython ''
```

Comment:
This should actually be done in the hook.

Reply:
I updated the PR. `python3Packages.gpiod` now is more like `python3Packages.sigrok`